### ISSUE TITLE
Error 500 when save Custom Field of type Select with Option Value empty

### ISSUE THREAD

**Issue by kimtiago** [mautic/mautic#10988]
### Mautic Version

4.2.x series

### PHP version

7.4.28

### What browsers are you seeing the problem on?

Chrome

### What happened?

Error 500 when save a new Custom Field of type Select with Option Value empty...
When save with option Value filled its Ok.

### How can we reproduce this issue?

Step 1: Click on Settings -> Custom Fields -> New
Step 2: Set any Label, Alias, object Contact and Group Core (i think isn't relevant)
Step 3: Set 'Data Type' to Select
Step 4: Click on 'Add a value' and fill Label
Step 5: Let 'Value' empty
Step 6: Click Save or Save & Close
 

### Relevant log output

```shell
mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Argument 2 passed to Mautic\CoreBundle\Helper\AbstractFormFieldHelper::appendChoice() must be of the type string, null given, called in /var/www/vhosts/xxxxxxx/public_html/mautic/app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php on line 236" at /var/www/vhosts/xxxxxxx/public_html/mautic/app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php line 247 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Argument 2 passed to Mautic\\CoreBundle\\Helper\\AbstractFormFieldHelper::appendChoice() must be of the type string, null given, called in /var/www/vhosts/xxxxxxx/public_html/mautic/app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php on line 236 at /var/www/vhosts/xxxxxxx/public_html/mautic/app/bundles/CoreBundle/Helper/AbstractFormFieldHelper.php:247)"} []
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by RCheesley** [comment]
Thanks for the detailed bug report @kimtiago !

**Comment by robertology** [comment]
I was going to work on this but could not reproduce.

**Comment by kuzmany** [comment]
Probably fixed in 4.2.2 https://github.com/mautic/mautic/pull/11101
